docs/dashscope-docs/万相2.7-图像.md

```
@@ -0,0 +1,389 @@
+**核心功能**:
+- 文生图(`messages.content` 仅 `text`,无 `image`)
+- 图像编辑(`messages.content` 有 `image` + `text`)
+- 交互式编辑(在编辑基础上传 `bbox_list` 框选区域)
+- 组图生成(`enable_sequential=true`)
+- 多图参考(传多张 `image`,模型综合参考)
+
+## 同步接口
+
+### Endpoint
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)